CMakeLists.txt

```diff
@@ -233,10 +234,56 @@ endif()
 
 if("${CPACK_GENERATOR}" STREQUAL "DEB")
 include(packaging)
+
+#list of components to be included in the package
+set(CPACK_COMPONENTS_ALL 
+	toolfpgaconf
```

Comment:
Is this CPACK_COMPONENTS_ALL used somewhere?

Reply:
That variable is to specify what components need to be packaged in the debian packages.